.ci/updatecli.d/update-specs.yml

```diff
@@ -58,9 +67,11 @@ actions:
       description: |-
         ### What
         APM agent json schema automatic sync
+
         ### Why
         *Changeset*
-        * https://github.com/elastic/apm-data/commit/{{ source "sha" }}
+        * {{ source "pull_request" }}
+        * https://github.com/elastic/apm/commit/{{ source "sha" }}
```

Comment:
Is that intentional that we move from `apm-data` to `apm` here ?

Reply:
```suggestion
        * https://github.com/elastic/apm-data/commit/{{ source "sha" }}
```

Reply:
good catch! my bad!